redshift/resource_redshift_schema.go

```diff
@@ -42,6 +43,12 @@ func redshiftSchema() *schema.Resource {
 				Description: "Keyword that indicates to automatically drop all objects in the schema, such as tables and functions. By default it doesn't for your safety",
 				Default:     false,
 			},
+			"quota": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Description: "In megabytes, the maximum amount of disk space that the specified schema can use",
+				Default:     0,
```

Comment:
I am open to leaving this as-is, but I wonder if it would be better to provide a non-unlimited value by default? I suppose this is the pre-quota definitions case though.

Reply:
Given that this was the behavior before the addition, it seemed reasonable.

Reply:
Ah, good point.